src/manage-apps/go-market-docs/app-center-ux-guidelines-enterprise.markdown

```diff
@@ -115,7 +115,9 @@ IMPORTANT: Any administrator can connect to your application from the SAP Concur
 
 #### <a name="landing-page-sign-in-or-up"></a>Sign In or Sign Up
 
-You should use this flow if you have an online account registration form.
+This is not the most common option to use, but this flow can be used if you have an online account registration form and/or you support free trials.
```

Comment:
Phrase this in a positive rather than negative manner.

Reply:
Suggested text: "Use this option if you support online account registration or free trials."

Reply:
sounds good.